jazz_apigee-proxy-aws/README.md

```diff
@@ -0,0 +1 @@
+Common Lambda integration for Apigee 
```

Comment:
Let's update Readme. Can you explain the purpose of this service?

Reply:
updated